### ISSUE TITLE
ClickHouse produces an INSERT with an empty column list

### ISSUE THREAD

**Issue by mrigger** [sqlancer/sqlancer#223]
ClickHouse produced a syntactically-invalid test case (see https://travis-ci.com/github/sqlancer/sqlancer/jobs/390341131):

```sql
DROP DATABASE IF EXISTS database4TLPHaving;
CREATE DATABASE IF NOT EXISTS database4TLPHaving;
CREATE TABLE t0 (c0 String MATERIALIZED ('-1255315139') CODEC (NONE)) ENGINE = Memory() ;
INSERT INTO t0() VALUES (), ();
INSERT INTO t0() VALUES (), ();
INSERT INTO t0() VALUES (), ();
INSERT INTO t0() VALUES (), (), (), ();
INSERT INTO t0() VALUES (), ();
INSERT INTO t0() VALUES (), (), ();
INSERT INTO t0() VALUES (), ();
[...]
```

When I execute the `CREATE TABLE` locally (on an older ClickHouse version), I get the following error:

```
Code: 90, e.displayText() = DB::Exception: Empty list of columns passed (version 20.4.1.2742 (official build))
``` 
I'm a bit confused that the `CREATE TABLE` and many of the `INSERT`s are executed without an error. @qoega, do you have any ideas on what the issue could be?

**Comment by qoega** [comment]
I've checked it on current version and empty brackets are prohibited after table name. It should fail
```
INSERT INTO t0() VALUES (), ();

Expected one of: insert element, qualified asterisk, asterisk, compound identifier, identifier, COLUMNS matcher, list of elements, COLUMNS
```

I'm not sure why there is a syntax error in travis run

**Comment by qoega** [comment]
Fixed in #275
